src/client.ts

```diff
@@ -117,7 +117,7 @@ export class Client extends EventSpewer {
     switch (this.authType) {
       case AuthTypes.BOT: return 'Bot';
     }
-    return '';
+    return 'Bearer';

```

Comment:
Might be worth it making this a case of AuthTypes.USER while preserving the return an empty string on unknown case behaviour